diff --git a/backend/linux/helper/split_tunneling/process_monitor.cpp b/backend/linux/helper/split_tunneling/process_monitor.cpp
index c1251b36..4e4defc4 100644
--- a/backend/linux/helper/split_tunneling/process_monitor.cpp
+++ b/backend/linux/helper/split_tunneling/process_monitor.cpp
@@ -23,8 +23,8 @@ void ProcessMonitor::monitorWorker(void *ctx)
     struct __attribute__ ((aligned(NLMSG_ALIGNTO))) {
         struct nlmsghdr nl_hdr;
         struct __attribute__ ((__packed__)) {
-            struct cn_msg cn_msg;
             struct proc_event proc_ev;
+            struct cn_msg cn_msg;
         };
     } nlcn_msg;
 
@@ -246,8 +246,8 @@ bool ProcessMonitor::prepareMonitoring()
     struct __attribute__ ((aligned(NLMSG_ALIGNTO))) {
         struct nlmsghdr nl_hdr;
         struct __attribute__ ((__packed__)) {
-            struct cn_msg cn_msg;
             enum proc_cn_mcast_op cn_mcast;
+            struct cn_msg cn_msg;
         };
     } nlcn_msg;
 
diff --git a/client/engine/engine/networkdetectionmanager/networkdetectionmanager_linux.cpp b/client/engine/engine/networkdetectionmanager/networkdetectionmanager_linux.cpp
index e6694b75..c85bc45e 100644
--- a/client/engine/engine/networkdetectionmanager/networkdetectionmanager_linux.cpp
+++ b/client/engine/engine/networkdetectionmanager/networkdetectionmanager_linux.cpp
@@ -86,7 +86,7 @@ void NetworkDetectionManager_linux::updateNetworkInfo(bool bWithEmitSignal)
 QString NetworkDetectionManager_linux::getDefaultRouteInterface(bool &isOnline)
 {
     QString strReply;
-    FILE *file = popen("/sbin/route -n | grep '^0\\.0\\.0\\.0'", "r");
+    FILE *file = popen("/usr/bin/env route -n | grep '^0\\.0\\.0\\.0'", "r");
     if (file)
     {
         char szLine[4096];
diff --git a/client/main.cpp b/client/main.cpp
index 4cd755cf..ff3ae5d4 100644
--- a/client/main.cpp
+++ b/client/main.cpp
@@ -77,9 +77,9 @@ int main(int argc, char *argv[])
         if (count != -1) {
             path = dirname(result);
         }
-        QStringList pluginsPath;
-        pluginsPath << QString::fromStdString(path) + "/plugins";
-        QCoreApplication::setLibraryPaths(pluginsPath);
+        // QStringList pluginsPath;
+        // pluginsPath << QString::fromStdString(path) + "/plugins";
+        // QCoreApplication::setLibraryPaths(pluginsPath);
     #endif
 #endif
 
